.github/workflows/pythontest-linux.yml

```diff
@@ -27,15 +27,10 @@ jobs:
     - name: Test with pytest
       shell: bash -l {0}
       env:
-        GSHEETS_TOKEN: ${{ secrets.gsheets_token }}
-        TOKEN_LOC: token.pickle
-        SHEET_ID: ${{ secrets.SHEET_ID }}
-        JWT_SECRET: ${{ secrets.JWT_SECRET }}
+        JWT_SECRET: faketest
       run: |
-        # Emit pickled gsheets testing credentials from CI secret
-        python -c "import pickle; import base64; import os; tok=pickle.loads(base64.b64decode(os.getenv('GSHEETS_TOKEN'))); tfile = open('token.pickle', 'wb'); pickle.dump(tok, tfile)"
         pip install pytest
-        pytest .
+        pytest --ignore icubam/db/test_gsheets.py
```

Comment:
Instead if ignoring them via CLI, maybe to add something along the lines of,

```py
ci_tokens = all([os.environ.get(key, False)
                 for key in ['GSHEETS_TOKEN', 'SHEET_ID', 'TOKEN_LOC]])

@unittest.skipIf(not ci-tokens,
                 "GSHEETS_TOKEN env variable not set")
```
https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures

to the two tests that fail, as otherwise users who clone the repo, wouldn't know to exclude this tests.

This closes https://github.com/icubam/icubam/issues/108 BTW.

Reply:
Thanks for the comments! 

The code would go into the gsheets unit test itself, right? So maybe rename `ci_tokens` to `gsheets_tokens`?

Usually devs who have the code set up on their machine and run the gsheets tests have a `token.pickle` file pointed to by `TOKEN_LOC` (it is only the current CI pipeline that stuffs it in a secret, base64-encoded, to avoid putting secrets in the tree). So how about restricting the tests to TOKEN_LOC and SHEET_ID? That would let forks / users with a blank config avoid running the tests while still failing loudly when it is expected to run (and it would keep running in CI)?

Reply:
> The code would go into the gsheets unit test itself, right? So maybe rename ci_tokens to gsheets_tokens?

Yes, there. Up to you, just to avoid confusion with `GSHEETS_TOKEN` env variable.

> So how about restricting the tests to TOKEN_LOC and SHEET_ID? That would let forks / users with a blank config avoid running the tests while still failing loudly when it is expected to run (and it would keep running in CI)?

Sounds good.